demo/qat-auto-reset/qat-auto-reset-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942 4944 4946}
+DEVS=$(for pf in $ENABLED_QAT_PF_PCIIDS; do lspci -n | grep -e "$pf" | grep -o -e "^\\S*"; done)
+
+set_auto_reset() {
+  for dev in $DEVS; do
+    auto_reset_path="/sys/bus/pci/devices/0000:$dev/qat/auto_reset"
+    echo "Setting $dev\'s auto-reset $1" # on/off
+    echo $1 > "$auto_reset_path"
+  done
+}
+
+inject_error() {
```

Comment:
Umm, I agree that we can re-use the part of enabling auto_reset in the initcontainer..
But, to me, having the part of injecting error in the initcontainer somewhat does not make sense.

Reply:
My proposal was to just inject the error using the pod spec without adding any scripts for it in the image.

Reply:
Do you want me to include it in `qat-autoreset.sh` in #1852

And then run an initcontainer image using the pod spec?

Reply:
Any image will work (it could also be `imageutils.GetE2EImage(imageutils.BusyBox)`) if you just set command/args to do something like `find /sys/kernel/debug/ -path '*qat_*heartbeat/inject_error' -exec tee 1 '{}' \;`

Reply:
@mythi Thanks for the comment, but unfortunately this does not work.
you cannot use `tee` as I told to `inject_error` I do not know why, but only `echo 1 > inject_error` works.

Reply:
Ah well, `find /sys/kernel/debug/qat_*/heartbeat/ -name inject_error -exec sh -c 'echo 1 > {}' \;` will work